packages/kbn-spec-to-console/README.md

```diff
@@ -34,3 +34,6 @@ yarn spec_to_console -g "<ELASTICSEARCH-REPO-FOLDER>/x-pack/plugin/src/test/reso
 * Request bodies
 * Data fetched at runtime: indices, fields, snapshots, etc
 * Ad hoc additions
+
+### Updating the script
+The script only allows a known parameter type to be used in the REST API specs and will throw an `Unexpected type error` if a new parameter type is encountered. The types can be updated in the file `<KIBANA_REPO>/packages/kbn-spec-to-console/lib/convert/params.js`. 
```

Comment:
Can we turn this path into a link for easier access?

Reply:
Also I suggest rephrasing this for clarity to:

> You can update [packages/kbn-spec-to-console/lib/convert/params.js](https://github.com/elastic/kibana/blob/master/packages/kbn-spec-to-console/lib/convert/params.js) to add or change the default values for various parameter types, for example `int`, `double`, and `enum`. The script will throw an `Unexpected type error` if you update it with a parameter type that isn't defined by the REST API specs.

Though as I'm reading this a few questions come to mind:
1. What is a "parameter type" in this context? Is it related to query parameters? Or is a parameter something specific to the Console autocomplete definition DSL?
2. What is `params.js` doing? Is it defining default values for each parameter type, or am I misunderstanding the intention of this code?